app/controllers/schools/core_programme/materials_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class Schools::CoreProgramme::MaterialsController < Schools::BaseController
+  skip_after_action :verify_authorized
```

Comment:
Let's authorize update on the SchoolCohort

Reply:
`SchoolCohortPolicy` added